src/components/Quote.vue

```diff
@@ -145,49 +145,6 @@ export default {
 			return subtitle
 		},
 
-		shortenQuotedMessage() {
-			let cutAtChar = 200
```

Comment:
Wouldn't it still make sense to do something like this in case of very long messages? There is not much use in putting 64k chars into the html when only 2 lines are rendered

Reply:
I'm not sure... on the other side also executing all that logic for each every quote message (most of them are not going to be that long) could also slow things down.. What do you think?

Reply:
Well we can kill the logic now and just substring(0, 250) or something

Reply:
done